safety/scan/finder/file_finder.py

```
@@ -149,8 +149,17 @@
                             files[file_type.value] = set()
                         files[file_type.value].add(inspectable_file)
                         break
+
+                special_files = {'pyproject.toml', 'env.yml', 'env.yaml'}
+                if file_name in special_files:
+                    file_type = FileType(file_name)
+                    inspectable_file = Path(root, file_name)
+                    if file_type.value not in files or not files[file_type.value]:
+                        files[file_type.value] = set()
+                    files[file_type.value].add(inspectable_file)
             level += 1
```

Ruff

Use `enumerate()` for index variable `level` in `for` loop

(SIM113)